packit_service/worker/whitelist.py

```diff
@@ -214,15 +213,16 @@ def check_and_report(self, event: Optional[Any], project: GitProject) -> bool:
             account_name = event.github_login
             if not account_name:
                 raise KeyError(f"Failed to get account_name from {type(event)}")
-
-            if not self.is_approved(account_name):
-                logger.error(f"User {account_name} is not approved on whitelist!")
+            namespace = event.base_repo_namespace
+            if not (self.is_approved(account_name) or self.is_approved(namespace)):
```

Comment:
@TomasTomecek Do you remember why you added this only to
`if isinstance(event, (PullRequestEvent, PullRequestCommentEvent)):`
but not to
`if isinstance(event, IssueCommentEvent):` (few lines below) ?

See #308 & #309

Reply:
b/c anyone could then do `/packit propose-update` - in that branch we need to verify that the user is approved

Reply:
Not sure I understand, but since you've ACKed #309 I don't need the answer anymore.